package.json

```diff
@@ -20,7 +21,7 @@
     "form-data": "^4.0.0",
     "imghash": "^0.0.9",
     "mysql2": "^3.7.0",
-    "ocr-space-api-wrapper": "^2.3.0",
+    "ocr-space-api-wrapper": "^2.1.3",
```

Comment:
we are using a vendored version that allows error handling. Please make sure we indeed never import it and remove